test/integration/api/network.bats

```diff
@@ -60,6 +60,27 @@ function teardown() {
 	[ "${#lines[@]}" -eq 4 ]
 }
 
+# docker network ls --filter name returns networks that match with a provided name
```

Comment:
@dhiltgen can you check that the behavior described by this test is what is expected?